examples/example-spreadsheet-dragfill.html

```diff
@@ -0,0 +1,370 @@
+    </ul>
+      <h2>View Source:</h2>
+      <ul>
+      <li>
+        <a href="https://github.com/6pac/SlickGrid/blob/master/examples/example-spreadsheet-dragfill.html" target="_sourcewindow">
+          View the source for this example on Github
+        </a>
+      </li>
+      </ul>
+  </div>
+</div>
+
+<script src="https://cdn.jsdelivr.net/npm/sortablejs/Sortable.min.js"></script>
+<script src="sortable-cdn-fallback.js"></script>
+
+<script src="../dist/browser/slick.core.js"></script>
+<script src="../dist/browser/slick.interactions.js"></script>
+<script src="../dist/browser/slick.grid.js"></script>
+<script src="../dist/browser/slick.editors.js"></script>
+<script src="../dist/browser/plugins/slick.contextmenu.js"></script>
+<script src="../dist/browser/plugins/slick.autotooltips.js"></script>
+<script src="../dist/browser/plugins/slick.cellrangedecorator.js"></script>
+<script src="../dist/browser/plugins/slick.cellrangeselector.js"></script>
+<script src="../dist/browser/plugins/slick.cellcopymanager.js"></script>
+<script src="../dist/browser/plugins/slick.hybridselectionmodel.js"></script>
+
+<script>
+  var grid;
+  var data = [];
+  let cellSelectionModel;
+  var options = {
+    editable: true,
+    enableAddRow: true,
+    enableCellNavigation: true,
+    asyncEditorLoading: false,
+    autoEdit: true
+  };
+  var contextMenuPlugin;
+  var allColIdArr = [];
+  /*
+   * Depending on the value of Grid option 'editorCellNavOnLRKeys', us
+   * Navigate to the cell on the left if the cursor is at the beginning of the input string
+   * and to the right cell if it's at the end. Otherwise, move the cursor within the text
+   */
+  function handleKeydownLRNav(e) {
+    var cursorPosition = this.selectionStart;
+    var textLength = this.value.length;
+    if ((e.keyCode === Slick.keyCode.LEFT && cursorPosition > 0) ||
+      e.keyCode === Slick.keyCode.RIGHT && cursorPosition < textLength - 1) {
+      e.stopImmediatePropagation();
+    }
+  }
+
+  function handleKeydownLRNoNav(e) {
+    if (e.keyCode === Slick.keyCode.LEFT || e.keyCode === Slick.keyCode.RIGHT) {
+      e.stopImmediatePropagation();
+    }
+  }
+
+  function TestValidator(value, grid, rowIndex, colIndex) {
+    if (value.toString().length > 4) {
+      return {
+        valid: false,
+        msg: "This field has more than 4 chars"
+      };
+    } else {
+      return {
+        valid: true,
+        msg: null
+      };
+    }
+  }
+ 
+  function TextEditor(args) {
+    var input;
+    var defaultValue;
+    var scope = this;
+    var navOnLR;
+    this.args = args;
+
+    this.init = function () {
+      navOnLR = args.grid.getOptions().editorCellNavOnLRKeys;
+      input = Slick.Utils.createDomElement('input', { type: 'text', className: 'editor-text' }, args.container);
+      input.addEventListener("keydown.nav", navOnLR ? handleKeydownLRNav : handleKeydownLRNoNav);
```

Comment:
I'm trying to copy more of your code on my end and I found that this code isn't being executed at all because `keydown.nav` is not a valid JS event. For the history side of things, this probably came from when we had jQuery because it allowed passing namespace.

BTW, do you know what the `editorCellNavOnLRKeys` grid option does exactly? I never used it myself and don't have code that uses this option either